crates/frontend/src/components/table.rs

```diff
@@ -7,6 +7,57 @@ use leptos::*;
 use serde_json::{json, Map, Value};
 use superposition_types::SortBy;
 
+#[component]
+pub fn expandable_text(
+    value: String,
+    max_length: usize,
+    formatter: types::CellFormatter,
+    row: Map<String, Value>,
+    class_name: String,
+    is_expandable: bool,
+) -> impl IntoView {
+    let value = StoredValue::new(value);
+    let get_child = move |val: &str| (formatter)(val, &row);
```

Comment:
is this bracket needed here ?
is this some mandatory requirement here for the function to be used ?
```suggestion
    let get_child = move |val: &str| formatter(val, &row);
```

Reply:
Yes mandatory